File: Java/POOber/src/poober/UberX.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package poober;

/**
 *
 * @author dell
 */
public class UberX extends Car {
    // se require el constructor de la clase que estoy heredando
    
    String brand;
    String model;
    //metodo sencillo
    public UberX (String license, Account driver, String brand, String model){
    super (license, driver); // la palabra clave super 
    //para referirse al metodo constructor de la clase padre
    this.brand = brand;
    this.model = model;
    
   // super hace referencia a los datos de la super clase y this hace 
           // referencia a los atributos de la clase hija
    
    
    
    }
            
}

File: PHP/UberX.php
<?php
require_once ('Car.php');
    class UberX extends Car {
    public $brand;
    public $model;


    Public function __construct ($license, $driver, $brand, $model){
        # se utiliza la palabra parent en vez de super para asiganarle los atributos a la clase padre
        #luego va su constructor
        parent ::__construct($license, $driver)

        $this->brand = $brand;
        $this->model = $model;

    }
}


?>